data-raw/parameters_inital-vals.R

```diff
@@ -0,0 +1,55 @@
+# Generate the interal package data, the parameters and the default intial states.
+
+DIR <- here::here("data-raw")
+
+# MEND 2013 -----------------------------------------------------------------------------------------
+# Wang, G., Post, W.M. and Mayes, M.A. (2013), Development of microbial‐enzyme‐mediated decomposition
+# model parameters through steady‐state and dynamic analyses. Ecological Applications,
+# 23: 255-272. doi:10.1890/12-0681.1
+
+MEND_params <- read.csv(file.path(DIR, "MEND_params.csv"), stringsAsFactors = FALSE)
+usethis::use_data(MEND_params, overwrite = TRUE, internal = FALSE)
+
+
+# Inital state values
+# Define the inital state values, these values are taken directly from the manuscript.
+B = 2
+D = 1
+P = 10
+Q = 0.1
+M = 5
+EP = 0.00001
+EM = 0.00001
+IC = 0
+Tot = 18.10002
+
+MEND_initalState <- c(P = P,  M = M,  Q = Q,  B = B,  D = D,  EP = EP,  EM = EM,  IC = IC,  Tot = Tot)
+usethis::use_data(MEND_initalState, overwrite = TRUE, internal = FALSE)
+
+# 2 Pool MEND  -----------------------------------------------------------------------------------------
+# Citation?
```

Comment:
@jianqiuz where the "default" values for the MEND2 model configuration come from?

Reply:
I will look into this